bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/util/Rows.scala

```diff
@@ -1,15 +1,16 @@
 package edu.gemini.pit.ui.util
 
 import com.jgoodies.forms.factories.Borders.DLU4_BORDER
-import swing.{Separator, GridBagPanel, Component, Label}
+
+import swing.{Component, GridBagPanel, Insets, Label, Separator}
 
 trait Rows { this: GridBagPanel =>
 
   border = DLU4_BORDER
   private var row = 0
-  def addRow(a: Component, b: Component, f:GridBagPanel.Fill.Value = GridBagPanel.Fill.Horizontal, wy:Int = 0 ) {
+  def addRow(a: Component, b: Component, f:GridBagPanel.Fill.Value = GridBagPanel.Fill.Horizontal, wy: Int = 0, gw: Int = 1) {
     add(a, new Constraints { gridx = 0; gridy = row; ipadx = 10; ipady = 4; anchor = GridBagPanel.Anchor.NorthEast })
-    add(b, new Constraints { gridx = 1; gridy = row; fill = f; weightx = 1; weighty = wy })
+    add(b, new Constraints { gridx = 1; gridy = row; fill = f; weightx = 1; weighty = wy; gridwidth = gw })
```

Comment:
Note minor changes to this class to simplify adding rows and minimize UI work.

I will post a shot of the new UI momentarily.